src/services/services.ts

```diff
@@ -2403,6 +2403,19 @@ module ts {
                         getCompletionEntriesFromSymbols(filteredMembers, activeCompletionSession);
                     }
                 }
+                else if (getAncestor(previousToken, SyntaxKind.ImportClause)) {
+                    // cursor is in import clause
+                    // try to show exported member for imported module
+                    isMemberCompletion = true;
+                    isNewIdentifierLocation = true;

```

Comment:
Also, shouldn't isMemberCompletion be `previousToken.kind !== SyntaxKind.AsKeyword`?